packages/cloud_firestore/cloud_firestore/test/cloud_firestore_test.dart

```diff
@@ -167,7 +169,14 @@ void main() {
           default:
             return null;
         }
-      });
+      };
+
+      // Ensure the MethodChannel platforms are initialized
+      Firestore.platform;
```

Comment:
It feels unintuitive that this has side effects. What do you think about doing a method instead?